### ISSUE TITLE
Broken Pipe Error

### ISSUE THREAD

**Issue by ghost** [janinamass/scythe#1]
While executing scythe.py with the respective input files, following error occurs:

```
SKIP  14
SKIP 14
group [15]
SKIP  15
SKIP 15
group [16]
group [17]
group [18]
group [19]
Writing files

...

group [20]
group [21]
Process ConsumerProc-1:
Traceback (most recent call last):
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "py3/bin/scythe.py", line 277, in run
    r,R = next_task.call()
  File "py3/bin/scythe.py", line 301, in call
    startAt = self.startAt)
  File "py3/bin/scythe.py", line 256, in makeFasta
    r, R = ah.sl_ref(scoringDct = avd, sequenceDct = seqDct), str(g)
  File "/home/chris/Dokumente/Bachelor/scythe/py3/bin/algo/algomod.py", line 130, in sl_ref
    uncoll.remove(cmaxid)
KeyError
Traceback (most recent call last):
  File "/usr/lib/python3.5/multiprocessing/queues.py", line 247, in _feed
    send_bytes(obj)
  File "/usr/lib/python3.5/multiprocessing/connection.py", line 200, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/usr/lib/python3.5/multiprocessing/connection.py", line 398, in _send_bytes
    self._send(buf)
  File "/usr/lib/python3.5/multiprocessing/connection.py", line 368, in _send
    n = write(self._handle, buf)
BrokenPipeError: [Errno 32] Broken pipe
```

Depedencies, such as emboss version being 6.4.0.0 and the python modules listed in the documentary, are fulfilled. Operating system: Ubuntu 16.04 Xenial Xerus

**Comment by janinamass** [comment]
not sure why exactly but groups with loci from only  < 3 species cause problems. As a temporary workaround, remove them from your grp file ( eg  less mammals.grp|awk '$4 {print;} !$4 {}' > mammals_new.grp )

**Comment by janinamass** [comment]
should be fixed with 194cb473f2ccb6450214fe9d2dc5b52da64b8245
